libs/sdk-backend-bear/src/convertors/toBackend/FilterConverter.ts

```diff
@@ -89,11 +89,21 @@ const convertAbsoluteDateFilter = (filter: IAbsoluteDateFilter): IVisualizationO
     };
 };
 
+/**
+ * @internal
+ */
+export interface IConvertInsightOptions {
+    allowNullInFilters?: boolean;
```

Comment:
Just two notes:
- I would name it `allowNullValuesInAttributeFilters` so it is more specific what nulls and in which filter
- Please put here some explanation when we need it: metadata conversion false, true only when convertor is used in postMessage conversion that uses similar interface to bear metadata but are less strict because they are used same interface is used for Bear. Something in that regard. Feel free to describe it better :)

Otherwise, this looks good imho 👌